src/main/java/Controller/ClientController.java

```diff
@@ -0,0 +1,30 @@
+package Controller;
+
+import Model.Client;
+import Service.ClientService;
+import Service.ClientServiceImpl;
+import View.AddClientView;
+import View.GetClientView;
+
+import java.util.Scanner;
+public class ClientController {
+    Scanner scanner = new Scanner(System.in);
+    private ClientService clientService = new ClientServiceImpl();
+    private final AddClientView addClientView = new AddClientView();
+    private final GetClientView getClientView = new GetClientView();
+
```

Comment:
Addition of new public method like 'processClient' can be considered. It would give us opportunity to make addClient and getAllClients both private and call them from processClient - just note to thing about it - does not need to be implemented.